spec/system/avo/actions_spec.rb

```diff
@@ -1,187 +1,211 @@
-# require 'rails_helper'
+require 'rails_helper'
 
-# RSpec.describe 'Actions', type: :system do
-#   let!(:roles) { { admin: false, manager: false, writer: false } }
-#   let!(:user) { create :user, active: true, roles: roles }
+RSpec.describe 'Actions', type: :system do
+  let!(:user) { create :user }
 
-#   context 'index' do
-#     describe 'without actions attached' do
-#       let(:url) { '/admin/resources/teams' }
+  describe 'action visibility' do
+    context 'index' do
+      it 'finds an action on index' do
+        visit '/admin/resources/users'
 
-#       it 'does not display the actions button' do
-#         visit url
+        click_on 'Actions'
 
-#         expect(page).not_to have_text 'Actions'
-#       end
-#     end
+        expect(page).to have_link 'Dummy action'
+      end
+    end
 
-#     describe 'with actions attached' do
-#       let!(:roles) { { admin: false, manager: false, writer: false } }
-#       let!(:second_user) { create :user, active: true, roles: roles }
-#       let(:url) { '/admin/resources/users' }
+    context 'show' do
+      it 'does not find an action on show' do
+        visit "/admin/resources/users/#{user.id}"
 
-#       it 'displays the actions button disabled' do
-#         visit url
+        click_on 'Actions'
```

Comment:
When there aren't any actions visible for a view, I don't think the `Actions` button should render at all.

Reply:
It [should be hidden](https://github.com/avo-hq/avo/blob/main/app/components/avo/views/resource_index_component.html.erb#L4) when no actions.